documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/RoutableFactories50.java

```diff
@@ -583,86 +583,6 @@ protected boolean doEncode(DocumentReply obj, DocumentSerializer buf) {
         }
     }
 
-    public static class BatchDocumentUpdateMessageFactory extends DocumentMessageFactory {
```

Comment:
Public class in PublicApi package. Same goes here. You have to make sure nobody is using it before removing, or remove it on '7'.